mesa/space.py

```diff
@@ -101,6 +101,9 @@ def __init__(self, width: int, height: int, torus: bool) -> None:
         # Add all cells to the empties list.
         self.empties = set(itertools.product(*(range(self.width), range(self.height))))
 
+        # Neighborhood Cache
+        self._neighborhood_cache = dict()  # type: Dict[Any, List[Coordinate]]
```

Comment:
Since Python 3.5 has been removed from support, we no longer have to use type comments.

Reply:
#984